examples/apps/branding/app.py

```diff
@@ -0,0 +1,304 @@
+
+        Learn more about [panel-material-ui](https://panel-material-ui.holoviz.org/).
+        """
+
+
+state = State()
+
+example_buttons = pn.FlexBox(
+    *(
+        pmui.Button(
+            name=color.capitalize(),
+            color=color,
+            width=120,
+            on_click=lambda e, color=color: pn.state.notifications.info(
+                f"Clicked the {color.upper()} button"
+            ),
+        )
+        for color in pmui.COLORS[:-1]
+    )
+)
+github_link = pmui.Button(
+    name="GitHub",
+    color="default",
+    variant="outlined",
+    href="https://github.com/panel-extensions/panel-material-ui",
+    target="_blank",
+    icon="open_in_new",
+)
+docs_link = pmui.Button(
+    name="Docs",
+    color="default",
+    variant="outlined",
+    href="https://panel-material-ui.holoviz.org/",
+    align="end",
+    target="_blank",
+    icon="open_in_new",
+)
+
+drawer = pmui.Drawer(pn.Spacer(height=75), stop_server, anchor="right", size=300)
+open_drawer = drawer.create_toggle(
+    icon="settings", sizing_mode="fixed", color="light", styles={"margin-left": "auto"}
+)
+
+cards = pmui.Row(
+    pmui.Paper(
+        pn.pane.Image(
+            "https://cdn.britannica.com/94/192794-050-3F3F3DDD/panels-electricity-order-sunlight.jpg",
+            margin=0,
+            sizing_mode="stretch_both",
+        ),
+        "# Reports\n\nDownload our annual report and see all our reports and presentations",
+        width=300,
+        sizing_mode="fixed",
+        margin=10,
+    ),
+    pmui.Paper(
+        pn.pane.Image(
+            "https://cdn.britannica.com/94/192794-050-3F3F3DDD/panels-electricity-order-sunlight.jpg",
+            margin=0,
+            sizing_mode="stretch_both",
+        ),
+        "# Reports\n\nDownload our annual report and see all our reports and presentations",
+        width=300,
+        sizing_mode="fixed",
+        margin=10,
+    ),
+    sizing_mode="fixed",
+    align="center",
+    margin=25,
+)
+
+
+@pn.cache
+def get_data(n_categories: int, n_rows=100):
+    x = np.random.rand(n_rows)
+    y = np.random.rand(n_rows)
+
+    categories = [f"Category {i+1}" for i in range(n_categories)]
+    category = np.random.choice(categories, n_rows)
+
+    dataframe = pd.DataFrame({"x": x, "y": y, "category": category}).sort_values(
+        "category"
+    )
+
+    return dataframe
+
+
+def get_categorical_plot(dark_theme: bool = False):
+    df = get_data(n_categories=3, n_rows=30)
+    cmap = colors.get_categorical_palette(n_colors=3, dark_theme=dark_theme)
+    return df.hvplot.scatter(
+        x="x",
+        y="y",
+        size=75,
+        color="category",
+        cmap=cmap,
+        height=350,
+        responsive=True,
+        title="Categorical Plot",
+        tools=["fullscreen"],
+        legend="top_right",
+    ).opts(backend_opts={"plot.toolbar.autohide": True}, toolbar="above")
+
+
+def get_continous_plot(dark_theme):
```

Comment:
When I toggle the theme my app moves to the top again. Would be better if it kept to current position. Don't know if its because I recreate the plots because their `cmap` has to change.


https://github.com/user-attachments/assets/3d95a59f-3bac-44cf-8a7d-1dfe86145377

Reply:
Quite possible, could also be React's fault for re-rendering everything.